utils/automation/FoScanner/Scanners.py

```diff
@@ -157,6 +157,62 @@ def get_copyright_list(self, all_results: bool = False) \
       return copyright_list
     return False
 
+  def get_copyright_whole(self, all_results: bool = False) \
```

Comment:
The body of this function is very similar to `get_copyright_list()`. Any reason to define this new function?

Reply:
get_copyright_list() only returned the copyright "content" from the keyword_scanner,
while get_copyright_whole() returns the "content", along with the byte info "start", "end" as well.

Reply:
Don't you think in such case it is better to extend the `get_copyright_list()` to return more and reduce the result as and when needed?

Reply:
Yeah! I see it now. For other function as well, their is a lot of unnecessary code duplication.
Can we maybe use a flag with the functions like `whole` which will default to false and we can pass it as true when we need the whole information from it.